scss/_body_common.scss

```diff
@@ -15,32 +15,39 @@
 
 /// Small text
 @mixin oTypographySmall {
```

Comment:
I suppose if someone wanted a small version, they could use the oTypographySans(s) or similar variant. I'm also not sure if anyone should use `<small>` tags.
@alicebartlett What are your thoughts on this?